xla/tests/collective_ops_e2e_test.cc

```diff
@@ -813,6 +838,16 @@ ENTRY main.12 {
   // Custom Calls.
   CollectiveOpsCompareWindowedNonWindowed(kModuleReplicatedStr,
                                           /*disable_dot_merger=*/true);
+
+  // Verify the creation of FP8 GEMM Custom Calls on Hopper and newer
+  // architectures.
+  DebugOptions opts = GetDebugOptionsForTest();
+  opts.set_xla_gpu_threshold_for_windowed_einsum_mib(0);
+  opts.set_xla_gpu_multi_streamed_windowed_einsum(true);
+  opts.set_xla_gpu_graph_min_graph_size(200);
+  opts.set_xla_gpu_enable_triton_gemm(false);
```

Comment:
How is this related to triton gemms?

Reply:
The purpose of this test is to verify that gemm_rewriter.cc creates an FP8 cublasLT custom call. The Triton rewriter may attempt to alternatively rewrite this into a Triton GEMM, which would interfere with this objective.

Reply:
Can you add a comment?